### PR TITLE
[Chat]Update rush version

### PR DESCRIPTION
**Pull request by Leah-Xia-Microsoft**
# What

Update rush version from to 5.112.2 to 5.115.0

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: rush.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
-  "rushVersion": "5.112.2",
```
Comment (vhuseinova-msft) [inline rush.json:3]: Rush version should also be updated in the GitHub workflows

Comment (emlynmac) [reply rush.json:3]: They're no longer in the CI workflow; rush version is set by the `rush.json` file.
The other workflows should only need any rush version to bootstrap the workflow on the build machine

Comment (vhuseinova-msft) [reply rush.json:3]: What about these ones https://github.com/search?q=repo%3AAzure%2Fcommunication-ui-library%205.112.2&type=code ?

Comment (vhuseinova-msft) [reply rush.json:3]: `The other workflows should only need any rush version to bootstrap the workflow on the build machin`
I'm a little bit worried that we would skip those files and they will stuck with the currrent version

Comment (vhuseinova-msft) [reply rush.json:3]: Maybe we can just use the latest version in those workflows instead of the pinned one

Comment (emlynmac) [reply rush.json:3]: Maybe we can just remove the pinned versions in the other workflows @Leah-Xia-Microsoft ?

**Pull request closed without merging**